MAVProxy/tools/MAVExplorer.py

```diff
@@ -1176,6 +1183,38 @@ def cmd_paramchange(args):
         vmap[pname] = pvalue
     mestate.mlog.rewind()
 
+def cmd_logmessage(args):
+    '''show log message information'''
+    mlog = mestate.mlog
+    usage = "Usage: logmessage <help|download>"
+    if len(args) > 0:
```

Comment:
```suggestion
    if len(args) <= 0:
        print(usage)
        return

```

... save indenting / context complication

.... `logmessage list` to show fields which are understood, perhaps?  Or are in the intersection of what help can provide and what's in the log?  ("future enhancement" :-) )